packages/store/src/base.ts

```diff
@@ -2,6 +2,29 @@ import type { Page } from './workspace/index.js';
 import type { TextType } from './text-adapter.js';
 import { Signal } from '@blocksuite/global/utils';
 import type * as Y from 'yjs';
+import { z } from 'zod';
+
+const BlockSchema = z.object({
+  version: z.number(),
+  schema: z.object({
+    flavour: z.string(),
+    tag: z.string(),
+    type: z.string().optional(),
+    defaultProps: z.record(z.any()),
+  }),
+});
+
+const CodeBlockSchema = {
```

Comment:
I think we should not put the per-block definition in the `store` package.

Reply:
just a example here